ecosystem/gstreamer_plugin/gst_mtl_st20p_rx.h

```diff
@@ -61,19 +61,15 @@ struct _Gst_Mtl_St20p_Rx {
   GstBuffer* buffer;
 
   /*< private >*/
-  struct st20p_rx_ops ops_rx;
-  gboolean silent;
+  struct st20p_rx_ops ops_rx; /* needed for caps negotiation */
```

Comment:
is ops_rx needed in this struct? can it be local to `_start()`? this applies to all such structs

Reply:
never mind. it is needed for negotiation... as comment says